db/seeds.d/61-foreman_tasks_bookmarks.rb

```diff
@@ -0,0 +1,16 @@
+class AddForemanTasksBookmarks < ActiveRecord::Migration
+  def up
+    Bookmark.where(:name => "running",
+                   :query => "state = running",
+                   :controller => "foreman_tasks_tasks",
+                   :public => true).first_or_create
+    Bookmark.where(:name => "failed",
+                   :query => "state = paused || result = error || result = warning",
+                   :controller => "foreman_tasks_tasks",
+                   :public => true).first_or_create
+  end
+
+  def down
+    Bookmark.where(:controller => "foreman_tasks_tasks").map(&:destroy)
```

Comment:
This would destroy user-created bookmarks too, are we ok with it?
Wouldn't `#each` be more suited here, since we don't want to keep the results?